File: midolman/src/main/scala/com/midokura/midolman/topology/rcu/PortSet.scala
/*
* Copyright 2012 Midokura Europe SARL
*/
package com.midokura.midolman.topology.rcu

import collection.immutable
import java.util.UUID


case class PortSet(id: UUID, hosts: immutable.Set[UUID],
                   localPorts: immutable.Set[UUID])

File: midolman/src/main/java/org/midonet/cluster/data/PortSet.java
/*
* Copyright 2012 Midokura Europe SARL
*/
package org.midonet.cluster.data;

import java.util.UUID;

public class PortSet extends Entity.Base<UUID, PortSet.Data, PortSet>{

    public PortSet(UUID uuid) {
        super(uuid, new Data());
    }

    @Override
    protected PortSet self() {
        return this;
    }

    public static class Data {}

}
